docs/create-new-credential-registry.md

```diff
@@ -71,12 +64,9 @@ If you made any changes to core code, you may have to manually resolve conflicts
 
 ### Customizing the User Interface
 
-The default theme and code is under the "default" theme.
-
-If you add a custom theme it will look there first.
-
-It follows standard Angular practices ...
-
+The client application is  a basic Angular application
+that follows standard Angular practices. You can make
+changes as you see fit.
```

Comment:
Should we remove this section altogether since we do not have a UI anymore?

Reply:
Nice catch.  I would prefer if it was updated to reference the template front-end [bcgov/aries-vcr-client](https://github.com/bcgov/aries-vcr-client) and the two concrete implementations as reference, [bcgov/orgbook-bc-client](https://github.com/bcgov/orgbook-bc-client), and [bcgov/orgbook-on-client](https://github.com/bcgov/orgbook-on-client).

When this does get contributed to Hyperledger the template front-end will go with it, but it will still be nice for developers to have references to some real world implementations.

@amanji, Would you be able to take on this document update?